src/components/side-panel/layer-panel/layer-panel-header.tsx

```diff
@@ -138,15 +139,21 @@ const HeaderActionSection = styled.div<HeaderActionSectionProps>`
   }
 `;
 
+type StyledPanelHeaderHiddenActionsProps = {
+  isConfigActive: LayerPanelHeaderProps['isConfigActive'];
+};
+
 // Hiden actions only show up on hover
 const StyledPanelHeaderHiddenActions = styled.div.attrs({
   className: 'layer-panel__header__actions__hidden'
-})`
+})<StyledPanelHeaderHiddenActionsProps>`
   opacity: 0;
   display: flex;
   align-items: center;
-  background-color: ${props => props.theme.panelBackground};
-  transition: ${props => props.theme.transition};
+  background-color: ${props =>
+    props.isConfigActive ? props.theme.panelBackgroundHover : props.theme.panelBackground};
+  transition: opacity 0.4s ease, background-color 0.4s ease;
+
```

Comment:
@heshan0131 these lines may need extra attention since there are theming improvements, but this upstreaming introduced some specific conditional formatting.

Reply:
looks for to me. I made it so the transition only applies to opacity and background color

Reply:
Makes sense, thanks for taking a look. Do you feel this PR is ready to go?